lib/Doctrine/Common/DataFixtures/Purger/ORMPurger.php

```diff
@@ -163,6 +169,8 @@ public function purge()
                 $connection->executeStatement($platform->getTruncateTableSQL($tbl, true));
             }
         }
+
+        $this->enableForeignKeyChecksForMySQL($connection);
```

Comment:
If anything inside the foreach throws, we leave the connection in a state of disabled foreign key checks, don't we? We should do something about that.